molviewspec/app/api/examples.py

```diff
@@ -1596,7 +1613,7 @@ async def portfolio_pdbekb_ligand_superpose(chains: str = "1tqn:A,2nnj:A") -> MV
         struct = builder.download(url=structure_url1).parse(format="mmcif").model_structure()
         if i > 0:  # this is just an example, transform will have to be different for each structure, of course
             struct.transform(
```

Comment:
Could have `scaling` here as well.

Reply:
Indeed, that could be a nice general option. Maybe everything should support `translation`, `rotation`, and `scaling`?

Reply:
> Indeed, that could be a nice general option. Maybe everything should support `translation`, `rotation`, and `scaling`?

I do agree. This will help to make the view more customizable. E.g. one can e.g., define two similar structures/meshes with the same parameters and position, and then `translate` them symmetrically to e.g. left and right, followed by `rotation` such they are mirrorred. That is probably much more convenient than try to hand-guess the positions and other params for them such that they will be placed symmetrically. @dsehnal, what do you think?

Reply:
No scaling for structures, needs to rotation + translation matrix.